VERSION.in

```diff
@@ -1 +1 @@
-0.17
+0.17.1
```

Comment:
This file shouldn't include a patch number, just major/minor.

I see the proto change that triggered this is just a comment update, so I think you can revert the change in this file and we can just ignore the check that fails as a result.

Reply:
```suggestion
0.17
```